docs/recipes/developing-and-testing/testing-the-client.md

```diff
@@ -104,53 +107,127 @@ Add a file called **index.html** to the root of the test project and add the fol
 </html>
 ```
 
-#### 6. Add test webpack config
+#### 7. Add test webpack config
 
-- Add a new file root directory called **webpack.tests.config.js**.
+- Add a new file root directory called **webpack.tests.config.js** with the following content
 
-- Populate it with the contents of a Fable-compatible webpack config template [such as this](https://github.com/fable-compiler/webpack-config-template/blob/master/webpack.config.js).
+```js
+// Template for webpack.config.js in Fable projects
+// Find latest version in https://github.com/fable-compiler/webpack-config-template
 
-#### 7. Update the test webpack config
+// In most cases, you'll only need to edit the CONFIG object (after dependencies)
+// See below if you need better fine-tuning of Webpack options
+
+// Dependencies. Also required: core-js, @babel/core,
+// @babel/preset-env, babel-loader, sass, sass-loader, css-loader, style-loader, file-loader, resolve-url-loader
+var path = require('path');
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+var CopyWebpackPlugin = require('copy-webpack-plugin');
 
-- Replace the `CONFIG` value in the webpack file you just created with the following:
-```fsharp
 var CONFIG = {
-    indexHtmlTemplate: './src/Client.Tests/index.html',
-    fsharpEntry: './src/Client.Tests/Client.Tests.fsproj',
-    outputDir: './src/Client.Tests',
-    assetsDir: './src/Client.Tests',
+    // The tags to include the generated JS and CSS will be automatically injected in the HTML template
+    // See https://github.com/jantimon/html-webpack-plugin
+    indexHtmlTemplate: 'src/Client.Tests/index.html',
+    fsharpEntry: 'src/Client.Tests/Library.fs.js',
+    outputDir: 'src/Client.Tests',
+    assetsDir: 'src/Client.Tests',
     devServerPort: 8081,
+    // When using webpack-dev-server, you may need to redirect some calls
+    // to a external API server. See https://webpack.js.org/configuration/dev-server/#devserver-proxy
     devServerProxy: undefined,
     babel: undefined
 }
-```
-
-- Remove all references and code from the webpack file that refers to MiniCssExtractPlugin or CONFIG.cssEntry.
 
-#### 8. Add launch command
-
-Add the following entry to the `scripts` element in the Client project's `package.json` file:
+// If we're running the webpack-dev-server, assume we're in development mode
```

Comment:
Is this all new content in the webpack file? If it's just a few lines that have changed, prefer instructions for that.

Reply:
This is showing how to add testing to the min template, which has no webpack.